int16[] velocities
int32[] positions
int8[] timeDiffs
time stamp
int16[] TEST_curTarget
